### PR TITLE
Add foosbot

### PR DESCRIPTION
**Pull request by jlelic**

### CODE REVIEW

File: backend/bot/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  }
+  winningTeam.forEach(player => {
+    if (player.name !== 'Pepa' && player.name !== 'Tonda') {
+      isComedyDuo = false;
+    }
+  })
+
+  let prefix = '';
+  if (isComedyDuo) {
+    prefix = ':tondab: :pepadab: ';
```
Comment (petrk-salsita) [inline backend/bot/index.js]: Move the trailing space to the messageText? (so that it doesn't need to be repeated)

File: backend/bot/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  } else if (ratingChange < 10) {
+    prefix = 'Easy. ';
```
Comment (petrk-salsita) [inline backend/bot/index.js]: Same

File: backend/bot/index.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  } else if (ratingChange > 20) {
+    prefix = 'HOLY SHIT!';
+  }
+
+  let suffix = '';
```
Comment (petrk-salsita) [inline backend/bot/index.js]: Use `const` and single expression

File: backend/repositories/match-repository.js [hunk 4]
```diff
@@ -79,5 +80,6 @@ const constructMatch = async (matchDescription) => {
   */
 exports.recordMatch = async (matchDescription) => {
     const match = await constructMatch(matchDescription)
+    bot.postResultToSlack(match);
```
Comment (petrk-salsita) [inline backend/repositories/match-repository.js]: Wait until the match is stored (there can be exceptions thrown) and post afterwards.

File: backend/bot/index.js (hunk 1 above)
Comment (jlelic) [reply backend/bot/index.js]: Then there would be empty space at the start of the `messageText` if the prefix ends up being empty.

Comment (petrk-salsita) [reply backend/bot/index.js]: Makes sense... then add it to 'HOLY SHIT!', where it's currently missing please 😄

File: backend/bot/index.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
+var SlackBot = require('slackbots');
+
+var bot = new SlackBot({
+  token: process.env.FOOSBOT_TOKEN
+});
+
+let channel;
+
+console.log('lez go');
```
Comment (novellizator) [inline backend/bot/index.js]: delete comment

File: backend/bot/index.js (hunk 1 above)
Comment (petrk-salsita) [reply backend/bot/index.js]: Oh, it's already there... nevermind!

**Pull request merged**